assets/app/view/game/part/track.rb

```diff
@@ -15,6 +15,21 @@ class Track < Snabberb::Component
           color: '#000000',
           width: 8,
           dash: '0',
+          broad: {
```

Comment:
How does this interact with user settings?

Reply:
User settings can override this.

Reply:
It's probably something to fix as part of a separate PR to allow narrow and dual to be overriden individually.